### ISSUE TITLE
Batch Normalization , Instance Normalization 和 Layer Normalization的实现

### ISSUE THREAD

**Issue by DamonsJ** [alibaba/MNN#595]
有没有这三个算子的实现计划？包括Metal的

**Comment by jxt1234** [comment]
Batch Normalization 本身是支持的

**Comment by jxt1234** [comment]
其他两个由 tensorflow 转过来默认是支持的

**Comment by DamonsJ** [comment]
sorry, 能具体一些吗？没有找到呢

**Comment by jxt1234** [comment]
可以看一下 
tools/converter/source/optimizer/postconvert/MergeBNToConvolution.cpp
tools/converter/source/optimizer/postconvert/TransformBatchNormal.cpp

Tensorflow 的模型里面没有单独的 Instance Normal 和 Layer Normalization Op，是由加减乘除等算符拼接出来的

**Comment by gangeqian** [comment]
那 除Pytorch 的Instance Normal ，Layer Normal, Group Normal 是否支持呢？
